src/accessControl/acp/index.js

```diff
@@ -63,6 +67,26 @@ const acpMapForCustomApplyPolicies = {
   addOnly: createAcpMap(false, false, true),
 };
 
+export const getPolicyNameFromAccess = (access) => {
+  const { read, write, append } = access;
+  if (read && write && !append) {
+    return "editors";
```

Comment:
All strings should be exported constants so that they can be used easily in tests, and so that if the string values are used multiple times they all come from a single definition.